sdk/dapp-kit/src/errors/walletErrors.ts

```diff
@@ -20,3 +20,13 @@ export class WalletAlreadyConnectedError extends Error {
 		Object.setPrototypeOf(this, WalletAlreadyConnectedError.prototype);
 	}
 }
+
+/**
+ * An error that is instantiated when someone attempts to perform an action that requires an active wallet connection.
+ */
+export class WalletNotConnectedError extends Error {
+	constructor(message: string) {
+		super(message);
+		Object.setPrototypeOf(this, WalletNotConnectedError.prototype);
```

Comment:
Is this _actually_ needed? I saw this historically but I'm not super convinced this is actually useful anymore

Reply:
I remember you asking this before on a PR haha... I guess the TypeScript docs I read were outdated because this example seems to work correctly with or without the manual prototype setting:

https://www.typescriptlang.org/play?#code/MYGwhgzhAEDqYhAUwC4DkD2KDCGB2eSwKSAJgKIBOlGl0SAHiXqTFTXQN4BQAkMPggpKAV2K0AFAFskUMAHMkALmhDKASzzyAlNB69eEEQAcklabIgKk2gNx9eAekfQA8gCMAVkRQA6CKgACjQoWACepq4AZhIoABbqEAA0cAjI6Fi4BD5k7LS+xiHhpnZ8AL7cFdzCYXrc0NDxNADu0ISt8IiomDj4hMS51JIA5CRCw6Vl0MBgKMBx0BI2dQ3qUYtI0JpCYHjASBjrnek9Wf0kFEOUujwNDQJ4EBjIviAY8hIARLCuAEoA0gBlT6lBpTJAgAIrO4PJ4vN4fT6AsCkEH1aAVMpAA